docs/development.rst

```diff
@@ -398,7 +398,130 @@ Common resources that a test might need,
 like e.g. a running MSColab server or a QApplication instance for GUI tests,
 are collected in :mod:`tests.fixtures` in the form of pytest fixtures that can be requested as needed in tests.
 
+Testing Keyring Features
+========================
```

Comment:
This will give a H1 section, on same level as the topic of the Page "Development"

The heading hierarchy is described at line 300 ... 

Please make it a H2 and change the others accordingly

Reply:
When you render by `make html ` with recent sphinx and sphinx_rtd_theme the current result is and shows the problem

![image](https://github.com/user-attachments/assets/24e32e91-e740-4b34-8fd7-32f573301628)

Reply:
and when you click on that, the whole document gets a different meaning